gluon/dal.py

```diff
@@ -117,3 +117,11 @@ def _default_validators(db, field):
         DRIVERS['pg8000'] = pg8000
     except:
         pass
+if not DRIVERS.get('sqlite3'):
```

Comment:
I think I can revert it and then make the change locally and add it to a .gitignore file or something.